README.md

```diff
@@ -1,2 +1,19 @@
 # Greetings-Microverse
```

Comment:
## Optional
- Please I suggest that you update your Readme and make it looks like the one in the template provided to you by Microverse. Open this link to read more: [README template](https://github.com/microverseinc/readme-template/blob/master/README.md)